src/app/_components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Button, TextField, Typography } from "@mui/material";
+import { useState } from "react";
+import type { ChangeEventHandler } from "react";
+
+// Defined interfaces for props
```

Comment:
this comment can be removed as well

Reply:
It should be named `LoginFormProps` or `ILoginForm` but something other than `LoginForm`